### PR TITLE
Stein Variational Gradient Descent Toy Example

### PR DESCRIPTION
**Pull request by ewfuentes**

### CODE REVIEW

File: experimental/svgd/svgd_sandbox.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    #     fig = plt.figure()
+    #     plt.contour(xs, ys, torch.exp(log_probs))
+    #     plt.colorbar()
+    #     plt.title(f'Step {i} Rev KL: {reverse_kl(particle_steps[i], obs_model)}')
+    #     plt.scatter(particle_steps[i, :, 0], particle_steps[i, :, 1])
+    #     plt.xlim(0, 10)
+    #     plt.ylim(0, 10)
+    #     plt.savefig(f'/tmp/particle_filter_{i:05d}.png')
+
+    #     plt.close()
```
Comment (ewfuentes) [inline experimental/svgd/svgd_sandbox.py:155]:
```suggestion
    for i in tqdm.tqdm(range(particle_steps.shape[0])):
        fig = plt.figure()
        plt.contour(xs, ys, torch.exp(log_probs))
        plt.colorbar()
        plt.title(f'Step {i} Rev KL: {reverse_kl(particle_steps[i], obs_model)}')
        plt.scatter(particle_steps[i, :, 0], particle_steps[i, :, 1])
        plt.xlim(0, 10)
        plt.ylim(0, 10)
        plt.savefig(f'/tmp/particle_filter_{i:05d}.png')

        plt.close()
```
